CONTRIBUTING.md

```diff
@@ -132,8 +137,37 @@ meantime.
 
 It can be helpful to go through older bug reports and make sure that they are
 still valid. Load up an older issue, double check that it's still true, and
-leave a comment letting us know if it is or is not. The [least recently updated sort][lru] is good for finding issues like this.
+leave a comment letting us know if it is or is not. The [least recently
+updated sort][lru] is good for finding issues like this.
+
+Contributors with sufficient permissions on the Rust repo can help by adding
+labels to triage issues:
+
+* Yellow, **A**-prefixed labels primarily relate to what part of the project
+  an issue is **about**. 
+
+* Magenta, **B**-prefixed labels identify bugs which **belong** elsewhere. 
+
+* Green, **E**-prefixed labels flag **easy** or introductory bugs. 
+
+* Red, **I**-prefixed labels warn of serious **issues** with the language
+  itself, such as regressions and slowdowns.
+
+* Orange, **P**-prefixed labels indicate a bug's **priority**. 
+
+* Blue, **T**-prefixed bugs denote where in the **toolchain** the issue fits.
```

Comment:
Ah these were actually recently added for the subteams actually :). There's a label-per-subteam in the RFC and rust repos.